grudge/discretization.py

```diff
@@ -1015,4 +1017,44 @@ def make_discretization_collection(
 # }}}
 
 
+# {{{ filter_part_boundaries
+
+def filter_part_boundaries(
```

Comment:
`get_boundary_dofdescs`? ("Filter" is possibly confusing with filtering-for-dissipation. And the function name didn't give me a vibe of returning `DOFDesc`s.)